modules/imaging_uploader/test/TestPlan.md

```diff
@@ -13,24 +13,57 @@
    [Manual Testing]      
 5. Ensure that the 'Upload' tab has the CandID, PSCID, and Visit Label greyed out upon loading. These greyed out fields remain
    as such if the 'Phantom Scans' is set to Yes, and become fillable and required fields if 'Phantom Scans' is set to No.  
-   [Manual Testing]      
-6. Try uploading an invalid file (a .jpg or .txt) with syntactically correct CandID, PSCID and visit labels. Make sure
-   an appropriate message is displayed in the Console Output at the top of the page.
    [Manual Testing]
-7. Execute step 6 again, but with a .tgz and .zip file.
+6. Test validation errors for 'CandID', 'PSCID', and 'Visit Label' in the 'Upload' tab and make sure that the appropriate
+   error messages are displayed under the relevant form elements.
+
+   Set Phantom Scans to 'No' and try to upload a valid .zip file for each scenario below:
+   - A 'CandID' that does not exist in the 'candidate' database (e.g. try appending letters to an existing CandID)
+   - A 'PSCID' that is invalid for a specific CandID in the 'candidate' database
+   - A 'Visit Label' that is invalid for a specific CandID in the 'session' database
+   - A candidate that has Active set to 'N' in the 'candidate' database
+   - A session that has Active set to 'N' in the 'session' database
+   - You should also consider scenarios where there are a combination of these errors or no errors at all
+
+   _e.g. SQL command to set candidate associated with CandID X as inactive:_
+
+   `UPDATE candidate SET Active = 'N' WHERE CandID = X;`
+
+   _e.g. SQL command to set session associated with CandID X and Visit Label Y as inactive:_
+
+   `UPDATE session SET Active = 'N' WHERE CandID = X AND Visit_label = 'Y';`
+
+   [Manual Testing]
+
+7. Test validations for 'File to Upload' in the 'Upload' tab and make sure that the appropriate
+   error messages are displayed under the 'File to Upload' element.
+
+   Set Phantom Scans to "Yes" and try to upload a file for each scenario below:
+   - A file that is not of type .tgz or tar.gz or .zip (e.g. a .jpg or .txt)
+   - A file that exceeds the max upload size
+
+   Set Phantom Scans to 'No' and try to upload a file for each scenario below:
+   - A file that is not of type .tgz or tar.gz or .zip (e.g. a .jpg or .txt)
+   - A file that exceeds the max upload size
+   - A file that that does not match the PSCID_CandID_VisitLabel convention
+      (once CandID, PSCID, and Visit Label have been validated)
```

Comment:
rephrase to:
(once the PSCID, candID and Visit Label **fields of the form** have been validated)